Add informations about community packages/repositories for Linux (#323)
apple/swift-org-website: download/index.md

$ yum install swiftlang
```

##### Fedora RPM repository

```bash
sudo dnf install swift-lang
``` 

> [Here's link to Fedora Repository for more information](https://src.fedoraproject.org/rpms/swift-lang#)

Reviewer: ```suggestion
[Here's a link to Fedora Repository for more information](https://src.fedoraproject.org/rpms/swift-lang#)
```

{% include_relative _older-releases.md %}

#### Arch Linux

> On Arch Linux you can install Swift using AUR. You can either use a repackedged version of the Fedora RPM or get a native build for Arch Linux.

Reviewer: ```suggestion
On Arch Linux you can install Swift using AUR. You can either use a repackaged version of the Fedora RPM or get a native build for Arch Linux.
```

> [More information availble at Arch Linux Wiki](https://wiki.archlinux.org/title/Swift)

Reviewer: ```suggestion
You can find more information available at [Arch Linux Wiki](https://wiki.archlinux.org/title/Swift).
```

##### Install using AUR helper tools

> Since Swift has AUR packages you can install it using AUR helper tools like yay.

Reviewer: ```suggestion
Since Swift has AUR packages you can install it using AUR helper tools like `yay`:
```

```bash
yay -S swift-language
```
or
```bash
yay -S swift-bin
```

##### Native Arch Linux build using AUR

[Source](https://aur.archlinux.org/packages/swift-language)

To install it you will need git and base-devel packages installed.

```bash
git clone https://aur.archlinux.org/swift-language.git
cd swift-language
makepkg -si
```
After that you will be asked if you want to instal requiered dependencies. Answer yes and wait for the build to finish. By the way since its building the whole thing it will take a while.

#####  Repackaged Fedora RPM

> Can be installed the same way as the previous one. Just change the repository. Also this package is faster to get up and running

Reviewer: ```suggestion
Can be installed the same way as the previous one, and this package is faster to get up and running. Just change the repository like this:
```

```bash
git clone https://aur.archlinux.org/swift-bin.git
cd swift-language
makepkg -si
```

#### Ubuntu/Debian community build

> There's also a community made package for Ubuntu/Debian based distros that is easy to install.

Reviewer: ```suggestion
There's also a community-made package for Ubuntu/Debian based distros that is easy to install.
```

Reviewer: I still feel like this one should be hyphenated, unless I'm missing something?

> Just copy the commands below to add the repository and install Swift.
> 
> [More information about the community repository](https://www.swiftlang.xyz/)
> 
> ```bash
> sudo apt install -y curl
> curl -s https://archive.swiftlang.xyz/install.sh | sudo bash
> sudo apt install swiftlang
> ```
> 

Reviewer: ```suggestion
Just copy the commands below to add the repository and install Swift:

```bash
sudo apt install -y curl
curl -s https://archive.swiftlang.xyz/install.sh | sudo bash
sudo apt install swiftlang
```

More information about the community repository is available at [swiftlang.xyz](https://www.swiftlang.xyz/).
```

> 

Reviewer: ```suggestion

More information about the community repository is available at [swiftlang.xyz](https://www.swiftlang.xyz/).
```

> sudo apt install swiftlang

Reviewer 2: @parpok Would you be able to share the repo/sources for how this package is being built and distributed?

Author: @shahmishal 
Here's a [Swift forum post about the apt repo](https://forums.swift.org/t/new-swift-package-repository-for-ubuntu-debian-linux-distributions/50412)
There's also more information on the repository project [website](https://www.swiftlang.xyz/user-guide) with included [link to scripts code for adding the repository](https://gist.github.com/futurejones/5d85565b2a5669fd4f4f2a78b80fdac6)

Reviewer: This doesn't provide sources for the build and packaging scripts unfortunately, which makes it impossible to verify that these packages were built correctly or to understand what modifications were done to the original source.

Author: I used `pkgdiff` on both the official tar.gz file from swift.org and deb file from swiftlang.xyz repository (downloaded using ```apt download swiftlang```) and the result came that files were different. If anyone interested in auditing the difference I'm leaving the [report `pkgdiff` provided](https://gist.github.com/parpok/4da032f73ea93b584a96a24ce9e1e2dd).

Also if the community repo doesn't suit needs I can delete any mentions of it.

![Overview of the `pkgdiff` report provided in the gist ](https://github.com/apple/swift-org-website/assets/63200231/1430265c-8100-4ad1-98a3-966695202da1)

Reviewer 3: @shahmishal  @MaxDesiatov  As the creator and maintainer of the https://swiftlang.xyz repository I can verify that all packages have been built without any modifications to the original source. 
Where possible the packages are created from the original .tar.gz files from swift.org. 
OS versions that swift.org does not supply are created from https://github.com/futurejones/swift-arm64/releases.
As I also build and maintain these packages I can also verify that they are built from the original source.
Build logs from https://ci.swiftlang.xyz/ show that no patches currently are needed for builds on ubuntu/debian on amd64 or arm64.
I use https://github.com/jordansissel/fpm to create the .deb packages from the .tar.gz files. `fpm` does not change or modify any files contained in the .tar.gz files, it simply repackages them into .deb files.

@parpok `pkgdiff` can not be used to check for diffs between two different file types like .tar and .deb.

Reviewer 2: >@shahmishal do we list non-swift.org produced installers / artifacts on this page, or is it limited to the ones the swift.org produces?
We currently don't post installer or artifacts urls from outside of swift.org.
